### PR TITLE
closes #57, created more database objects

### LINKED ISSUE #57: #931
Tie curricula table and user table together

### PR DESCRIPTION
**Pull request by bammons**

### CODE REVIEW

File: app/models/curricula.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 class Curricula < ActiveRecord::Base
 
-	has_many :users
+    belongs_to :creator,      foreign_key: "creator_id", class_name: "User"
+
+    has_many :users_projects, dependent: :destroy
```
Comment (seth-marquez) [inline app/models/curricula.rb]: note: change to user_curricula

File: app/models/curricula.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+    has_many :users, through: :users_projects
```
Comment (seth-marquez) [inline app/models/curricula.rb]: note: change to users_curricula

**Comment by seth-marquez** [comment]
nah, fix yo things, make that shit work
-1

**Comment by kdorer** [comment]
+1

**Comment by Linkster700** [comment]
+1

**Comment by ChristianMurphy** [comment]
[+1 success](https://i.chzbgr.com/maxW500/8075658240/h0D06B2A6/)

**Pull request merged**